hosted/alibaba/helper/helper_cluster.go

```diff
@@ -85,6 +90,221 @@ func CreateAlibabaHostedCluster(client *rancher.Client, displayName, cloudCreden
 	return clusterResp, nil
 }
 
+func CreateALIClusterOnAlibaba(csClient *cs.Client, region string, clusterName string, k8sVersion string, nodes string, tags map[string]string, extraArgs ...string) (string, error) {
+
+	// Load aliClusterConfig from YAML config
+	var aliClusterConfig ali.ClusterConfig
+	config.LoadConfig(ali.ALIClusterConfigConfigurationFileKey, &aliClusterConfig)
+
+	if k8sVersion != "" {
+		aliClusterConfig.KubernetesVersion = k8sVersion
+	}
+
+	nodepool := &cs.Nodepool{
+		KubernetesConfig: &cs.NodepoolKubernetesConfig{
+			Runtime:        tea.String(aliClusterConfig.NodePools[0].Runtime),
+			RuntimeVersion: tea.String(aliClusterConfig.NodePools[0].RuntimeVersion),
+		},
+		NodepoolInfo: &cs.NodepoolNodepoolInfo{
+			Name:            tea.String(aliClusterConfig.NodePools[0].Name),
+			ResourceGroupId: tea.String(aliClusterConfig.ResourceGroupID),
+		},
+		ScalingGroup: &cs.NodepoolScalingGroup{
+			VswitchIds:         tea.StringSlice(aliClusterConfig.VSwitchIDs),
+			DesiredSize:        tea.Int64(aliClusterConfig.NodePools[0].DesiredSize),
+			InstanceTypes:      tea.StringSlice([]string{aliClusterConfig.NodePools[0].InstanceTypes[0]}),
+			SystemDiskCategory: tea.String(aliClusterConfig.NodePools[0].SystemDiskCategory),
+			SystemDiskSize:     tea.Int64(aliClusterConfig.NodePools[0].SystemDiskSize),
+			DataDisks: []*cs.DataDisk{
+				{
+					Category:  tea.String(aliClusterConfig.NodePools[0].DataDisks[0].Category),
+					Size:      tea.Int64(int64(aliClusterConfig.NodePools[0].DataDisks[0].Size)),
+					Encrypted: tea.String("false"),
+				},
+			},
+		},
+	}
+	req := &cs.CreateClusterRequest{
+		Name:                 tea.String(clusterName),
+		ClusterType:          tea.String(aliClusterConfig.ClusterType),
+		ClusterSpec:          tea.String(aliClusterConfig.ClusterSpec),
+		KubernetesVersion:    tea.String(k8sVersion),
+		ServiceCidr:          tea.String(aliClusterConfig.ServiceCIDR),
+		Vpcid:                tea.String(aliClusterConfig.VpcID),
+		VswitchIds:           tea.StringSlice(aliClusterConfig.VSwitchIDs),
+		PodVswitchIds:        tea.StringSlice(aliClusterConfig.PodVswitchIDs),
+		Nodepools:            []*cs.Nodepool{nodepool},
```

Comment:
nit:
If it is possible to create more than 1 nodepool, perhaps you should give that option in `CreateALIClusterOnAlibaba` function.